packages/geo/src/lib/filter/ogc-filterable-item/ogc-filterable-item.component.ts

```diff
@@ -165,7 +165,8 @@ export class OgcFilterableItemComponent implements OnInit {
 
   addFilterToSequence() {
     this.filtersCollapsed = false;
-    const arr = this.datasource.options.ogcFilters.interfaceOgcFilters;
+    const interfaceOgcFilters = this.datasource.options.ogcFilters.interfaceOgcFilters;
```

Comment:
What is interfaceOgcFilters? I see it's typed as any[] ?

Reply:
Also, OgcFilterWriter seems like a class with some utility functions and no state. Should all datasources really have their own instance created automatically? Maybe you coiuld instantiate one when needed instead of doing "(this.datasource as any).ogcFilterWriter".

Reply:
About OgcFilterWriter , included in a new branch/PR. I'll fix it soon.